ipalib/config.py

```diff
@@ -532,8 +533,12 @@ def _finalize_core(self, **defaults):
 
         # Merge in context config file and then default config file:
         if self.__d.get('mode', None) != 'dummy':
-            self._merge_from_file(self.conf)
-            self._merge_from_file(self.conf_default)
```

Comment:
I'd leave this failing with traceback, validation of input should happen in different place